pkg/driver/mount.go

```diff
@@ -293,17 +313,13 @@ func (n *NodeMounter) remountVolume(
 	vol *jv.JivaVolume,
 ) (err error) {
 	options := []string{"rw"}
-	// Wait until it is possible to change the state of mountpoint or when
-	// login to volume is possible
-	_, err = waitForVolumeToBeReady(vol.Name, n.client)
-	if err != nil {
-		return
-	}
 
-	err = waitForVolumeToBeReachable(fmt.Sprintf("%v:%v", vol.Spec.ISCSISpec.TargetIP,
-		vol.Spec.ISCSISpec.TargetPort))
-	if err != nil {
-		return
+	if ready, err := isVolumeReady(vol.Name, n.client); err != nil || !ready {
```

Comment:
Q: When will the JivaVolume phase and status will be updated?

Reply:
This will be updated only when 2 replicas are connected and in RW state